testsuite/pytests/sli2py_stimulating/test_iaf_ps_psp_poisson_generator_accuracy.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# test_iaf_ps_psp_poisson_generator_accuracy.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""
+Tests for correct voltage of precise timing neuron receiving input from precise timing poisson_generator
+
+The tests generates a poisson spike train using the poisson generator
+for precise spike times. In a second step this spike train is supplied
+to a neuron model and the resulting subthreshold membrane potential
+fluctuations are compared to the analytical solution.  Thus, in
+contrast to the more advanced test_iaf_ps_psp_poisson_accuracy, this
+test does not require the interaction of the generator and the neuron
+model to work and does not require the availability of a parrot
+neuron.  In contrast to test_iaf_ps_psp_poisson_accuracy the DC
+required to maintain a subthreshold membrane potential is generated by
+a dc generator not a property of the neuron model.  The
+spike_generator used to supply the neuron model with spikes,
+constraints spike times to the tic grid of the simulation kernel. This
+is the temporal resolution in which the computation step size and
+simulation times are expressed. Therefore, the results of simulations
+at different computation step sizes only differ because of limited
+machine precision.  The difference between the analytical result and
+the simulation, however, is dictated by the number of tics per
+millisecond.
+
+Author:  May 2005, February 2008, March 2009; Diesmann
+References:
+ [1] Morrison A, Straube S, Plesser H E, & Diesmann M (2007) Exact Subthreshold
+     Integration with Continuous Spike Times in Discrete Time Neural Network
+     Simulations. Neural Computation 19:47--79
+SeeAlso: testsuite::test_iaf_ps_psp_accuracy, testsuite::test_iaf_ps_dc_accuracy
+"""
+
+
+import math
+from math import exp
+
+import nest
+import pytest
+
+DEBUG = False
+
+# Global parameters
+T = 6.0
+tau_syn = 0.3
+tau_m = 10.0
+C_m = 250.0
+weight = 65.0
+delay = 1.0
+
+neuron_params = {
+    "E_L": 0.0,
+    "V_m": 0.0,
+    "V_th": 1500.0,
+    "I_e": 0.0,
+    "tau_m": tau_m,
+    "tau_syn_ex": tau_syn,
+    "tau_syn_in": tau_syn,
+    "C_m": C_m,
+}
+
+
+def alpha_fn(t):
```

Comment:
This is the change in membrane potential at `t2` due to an alpha-shaped input spike arriving at `t1`, isn't it? Maybe put that in a docstring.